inference-engine/src/transformations/src/transformations/rt_info/old_api_map_attribute.cpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/rt_info/old_api_map_attribute.hpp"
+
+using namespace ov;
+
+bool OldApiMap::visit_attributes(AttributeVisitor& visitor) {
+    visitor.on_attribute("order", m_value.m_order);
+    visitor.on_attribute("type", m_value.m_type);
```

Comment:
what is the default value for `type`? Will it be the same as port type if no differences between IR v10 and IR v11?

Reply:
The default value for `type` is still under discussion. @jane-intel proposed to extend nGraph serialization to make it possible to set type=None. I will discuss it with @GlebKazantaev and inform you about the the final decision.